### ISSUE TITLE
Specious warning to upgrade to 1.61.0 when already using 1.62.0

### ISSUE THREAD

**Issue by jeffrey-aguilera** [getsentry/sentry-cli#898]
### Environment

How do you use Sentry?
SaaS

Which SDK and version?
sentry-cli 1.62.0

### Steps to Reproduce

1. Run any sentry-cli command other than sentry-cli --version

### Expected Result

1. No version warning, as 1.62.0 is the latest version

### Actual Result

```
❯ sentry-cli releases -o my-organization list
...
sentry-cli update to 1.61.0 is available!
run brew upgrade sentry-cli to update

❯ sentry-cli --version
sentry-cli 1.62.0
```

**Comment by kamilogorek** [comment]
Can you provide logs using `SENTRY_LOG_LEVEL=debug`? The update logic is based on the binary stored in the cache directory, which seems to be out-of-sync.

**Comment by kamilogorek** [comment]
@jeffrey-aguilera your error should be gone now :)

**Comment by brfrth** [comment]
Still getting this one 1.63.

![image](https://user-images.githubusercontent.com/6641565/110481249-a6d96d80-80e7-11eb-844b-a296d5ec81f6.png)

**Comment by ThibaultVlacich** [comment]
Same, getting the issue with 1.63.1

```
$ sentry-cli info
Sentry Server: https://sentry.io
Default Organization: -
Default Project: -

sentry-cli update to 1.63.1 is available!
run sentry-cli update to update
$ sentry-cli --version
sentry-cli 1.63.1
```

**Comment by kamilogorek** [comment]
Whoops, forgot to cut a release after the patch 😅 Released as `1.63.2`.
